binder/src/main/java/io/grpc/binder/BinderChannelBuilder.java

```diff
@@ -216,6 +279,22 @@ public BinderChannelBuilder securityPolicy(SecurityPolicy securityPolicy) {
     return this;
   }
 
+/**
+   * Provides the target {@UserHandle} of the remote Android service.
+   *
+   * <p>When targetUserHandle is set, Context.bindServiceAsUser will used and additional Android
+   * permissions will be required. If your usage does not require cross-user communications, please
+   * do not set this field. It is the caller's responsibility to make sure that it holds the
+   * corresponding permissions.
+   *
+   * @param targetUserHandle the target user to bind into.
+   * @return this
+   */
+  public BinderChannelBuilder bindForUser(UserHandle targetUserHandle) {
```

Comment:
I'm not sure if we discussed this already, but why not "bindAsUser"? (Just to mirror the Android API really).

Reply:
Done.